Lib/test/test_types.py

```diff
@@ -1409,6 +1410,18 @@ class F(list[int]): pass
         self.assertEqual(types.get_original_bases(E), (list[T],))
         self.assertEqual(types.get_original_bases(F), (list[int],))
 
+        class FirstBase(typing.Generic[T]): pass
+        class SecondBase(typing.Generic[T]): pass
+        class First(FirstBase[int]): pass
+        class Second(SecondBase[int]): pass
+        class G(First, Second): pass
+        self.assertEqual(types.get_original_bases(G), (First, Second))
+
+        class First_(typing.Generic[T]): pass
+        class Second_(typing.Generic[T]): pass
+        class H(First_, Second_): pass
+        self.assertEqual(types.get_original_bases(G), (First, Second))
```

Comment:
I think this is meant to be

```suggestion
        self.assertEqual(types.get_original_bases(H), (First_, Second_))
```

Reply:
Yep, nice spot